website/www/site/layouts/partials/section-menu/en/documentation.html

```diff
@@ -225,6 +225,7 @@
     <li><a href="/documentation/ml/anomaly-detection/">Anomaly Detection</a></li>
     <li><a href="/documentation/ml/large-language-modeling">Large Language Model Inference in Beam</a></li>
     <li><a href="/documentation/ml/per-entity-training">Per Entity Training in Beam</a></li>
+    <li><a href="/documentation/ml/tensorrt-runinference">TensorRT Text Classification Inference</a></li>
```

Comment:
```suggestion
    <li><a href="/documentation/ml/tensorrt-runinference">TensorRT Inference</a></li>
```

This will help this render more naturally and illustrates the main point of the document